mmeval/metrics/end_point_error.py

```diff
@@ -122,13 +125,13 @@ def end_point_error_map(
         epe = epe_map.reshape(-1)[val]
         return epe.mean(keepdims=True), int(val.sum())
 
-    @dispatch
-    def end_point_error_map(
-            self,
-            prediction: 'torch.Tensor',
-            label: 'torch.Tensor',
-            valid_mask: Optional['torch.Tensor'] = None
-    ) -> Tuple[np.ndarray, int]:
+    @dispatch  # type: ignore
+    def end_point_error_map(  # type: ignore
+        self,
+        prediction: 'torch.Tensor',
+        label: 'torch.Tensor',
+        valid_mask: Optional['torch.Tensor'] = None
+    ) -> Tuple[np.ndarray, int]:  # type: ignore
```

Comment:
We should add `typing.overload` decorator to this function and remove unnecessary changes.
```suggestion
    @overload
    @dispatch  # type: ignore
    def end_point_error_map(
            self,
            prediction: 'torch.Tensor',
            label: 'torch.Tensor',
            valid_mask: Optional['torch.Tensor'] = None
    ) -> Tuple[np.ndarray, int]:
```